scripts/build/custom_platforms.js

```diff
@@ -658,10 +658,7 @@ export const CUSTOM_PLATFORMS_CONFIG = {
         'path': 'extension/chromium-mv3',
         'expires': '10 days',
         'configuration': {
-            'removeRulePatterns': [
-                ...CHROMIUM_BASED_EXTENSION_PATTERNS,
-                ...REDIRECT_MODIFIER_PATTERNS,
```

Comment:
Should not `REDIRECT_MODIFIER_PATTERNS` be replaced with `REDIRECT_RULE_MODIFIER_PATTERNS`?
If I'm not wrong, `redirect-rule` modifier is not supported by MV3 extension.
The same below.